scripts/globals/pasteventwatcher.lua

```diff
@@ -0,0 +1,220 @@
+-----------------------------------
+-- Past Event Watcher NPCs
+-----------------------------------
+
+-----------------------------------
+
+require("scripts/globals/quests")
+require("scripts/globals/missions")
+require("scripts/globals/zone")
+
+pasteventwatcher = {}
```

Comment:
It'd be nice if we kind if we followed what's become the "standard" of loading everything in to the DSP table.

```lua
dsp = dsp or {}
dsp.goblinfootprint = dsp.goblinfootprint or {}

etc
```

Reply:
fixed 😸